packages/esm-patient-registration-app/src/patient-registration/input/custom-input/identifier/identifier-input.component.tsx

```diff
@@ -138,6 +138,8 @@ export const IdentifierInput: React.FC<IdentifierInputProps> = ({ patientIdentif
         {!patientIdentifier.required && !defaultPatientIdentifierTypesMap[patientIdentifier.identifierTypeUuid] && (
           <UserHasAccess privilege="Delete Patient Identifiers">
             <Button
+              style={{ marginBottom: '1rem' }}
+              size="md"
```

Comment:
These changes will be applied to the above 2 buttons as well.
Just a minor change.
Thanks!